editor/renames_map_3_to_4.cpp

```diff
@@ -1060,40 +1016,49 @@ const char *RenamesMap3To4::csharp_function_renames[][2] = {
-	//  { "percent_visible, "show_percentage }, // ProgressBar -- Breaks Label and RichTextLabel. Could it be worth it?
-	//	{ "process_mode", "process_callback" }, // AnimationTree, Camera2D
-	//	{ "scancode", "keycode" }, // InputEventKey
-	//	{ "toplevel", "top_level" }, // Node
-	//	{ "window_title", "title" }, // Window
-	//	{ "wrap_enabled", "wrap_mode" }, // TextEdit
-	//	{ "zfar", "far" }, // Camera3D
-	//	{ "znear", "near" }, // Camera3D
-	//	{ "filename", "scene_file_path" }, // Node
-	//	{ "pressed", "button_pressed" }, // BaseButton -- Would also rename the signal, skipping for now.
+	// NOTE: Commented out renames are disabled because deemed not suitable for
+	// the current way the regex-based converter works.
+	// When uncommenting any of those as suitable for conversion, please move it
+	// to the block with other enabled conversions, ordered alphabetically, and
+	// make sure to add it to the C# rename map too.
+
+	// Too common words, users may use these names for variables or in comments.
+	// { "bg", "panel" }, // Theme
+	// { "alt", "alt_pressed" }, // InputEventWithModifiers
+	// { "command", "command_pressed" }, // InputEventWithModifiers
+	// { "control", "ctrl_pressed" }, // InputEventWithModifiers
+	// { "d", "distance" }, // WorldMarginShape2D
+	// { "device", "output_device" }, // AudioServer
+	// { "doubleclick", "double_click" }, // InputEventMouseButton
+	// { "filename", "scene_file_path" }, // Node
+	// { "group", "button_group" }, // BaseButton
+	// { "meta", "meta_pressed" }, // InputEventWithModifiers
+	// { "rotate", "rotates" }, // PathFollow2D
+	// { "off", "unchecked" }, // Theme
+	// { "ofs", "offset" }, // Theme
+	// { "offset", "progress" }, // PathFollow2D, PathFollow3D
+	// { "on", "checked" }, // Theme
+	// { "shift", "shift_pressed" }, // InputEventWithModifiers
+	// { "window_title", "title" }, // Window
+	// { "zfar", "far" }, // Camera3D
+	// { "znear", "near" }, // Camera3D
+
+	// Would need bespoke solution.
+	// { "autowrap", "autowrap_mode" }, // Label -- Changed from bool to enum.
+	// { "frames", "sprite_frames" }, // AnimatedSprite2D, AnimatedSprite3D -- GH-73696
+	// { "percent_visible, "show_percentage }, // ProgressBar -- Breaks Label and RichTextLabel.
+	// { "pressed", "button_pressed" }, // BaseButton -- Would also rename the signal.
+	// { "process_mode", "process_callback" }, // AnimationTree, Camera2D -- conflicts with Node.
+	// { "wrap_enabled", "wrap_mode" }, // TextEdit -- Changed from bool to enum.
+
 	{ "as_normalmap", "as_normal_map" }, // NoiseTexture
 	{ "bbcode_text", "text" }, // RichTextLabel
-	{ "bg", "panel" }, // Theme
 	{ "bg_focus", "focus" }, // Theme
 	{ "capture_device", "input_device" }, // AudioServer
 	{ "caret_blink_speed", "caret_blink_interval" }, // TextEdit, LineEdit
 	{ "caret_moving_by_right_click", "caret_move_on_right_click" }, // TextEdit
 	{ "caret_position", "caret_column" }, // LineEdit
+	{ "cast_to", "target_position" }, // RayCast2D, RayCast3D
```

Comment:
Seems safe enough? There's `Object::cast_to` but it's not part of the scripting API.